python/foxglove-sdk/src/schemas.rs

```diff
@@ -0,0 +1,1921 @@
+//! Definitions for well-known Foxglove schemas
+//! Generated by https://github.com/foxglove/schemas
```

Comment:
It would be nice if possible to put all the generated files in a subdirectory, rather than mixed in with normal code, so that we can easily clean it prior to regenerate.

In https://github.com/foxglove/foxglove-sdk/pull/202 I've started consistently doing this, I also found a bug in https://github.com/foxglove/foxglove-sdk/pull/203 as a result of not fully cleaning generated output directories.

Reply:
The generated rust files are now in a subdirectory. I'll clean this up a big once #202 merges.

I don't see an easy way to do the same for the .pyi stubs, since (I think) they need to fit into the expected module hierarchy, and that doesn't all come from codegen.